mod-fqm-manager/src/test/java/org/folio/QuerySimulation.scala

```diff
@@ -0,0 +1,41 @@
+package org.folio
+
+import com.intuit.karate.gatling.PreDef._
+import io.gatling.core.Predef._
+import org.apache.commons.lang3.RandomUtils
+
+import scala.concurrent.duration._
+import scala.language.postfixOps
+
+class OrderSimulation extends Simulation {
```

Comment:
Clearly this was copy/pasted 😆 
```suggestion
class QuerySimulation extends Simulation {
```